apps/sms/services/js/activity/client.js

```diff
@@ -0,0 +1,102 @@
+/* global EventDispatcher,
+          bridge
+*/
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for managing activities.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'activity-service';
```

Comment:
I have mixed feelings, if we're going to leave client and just replace shim with service hosted in ServiceWorker in the future, then I'd say we need to use ```service``` term, so that client doesn't notice any change, but if we're going to live with the shim for some long term maybe we can use ```shim``` prefix instead to make it clearer for developers.

Both options work for me, just let me know what you prefer.